Tests/SonarScanner.MSBuild.Common.Test/ProcessRunnerTests.cs

```diff
@@ -57,13 +57,13 @@ public void ProcRunner_ExecutionFailed()
 
         var logger = new TestLogger();
         var args = new ProcessRunnerArguments(exeName, true);
-        using var runner = new ProcessRunner(logger);
+        var runner = new ProcessRunner(logger);
 
         // Act
         var success = runner.Execute(args);
 
         // Assert
-        success.Should().BeFalse("Expecting the process to have failed");
+        success.Succeeded.Should().BeFalse("Expecting the process to have failed");
         runner.ExitCode.Should().Be(-2, "Unexpected exit code");
```

Comment:
For consistency, the `ExitCode` should also be part of the return value as well. This should be done in a follow-up PR, though.